tests/integration/docusaurus/setup/configuring_data_contexts/how_to_configure_credentials.py

```diff
@@ -2,183 +2,135 @@
 
 import great_expectations as gx
 from great_expectations.core.yaml_handler import YAMLHandler
-from great_expectations.datasource.new_datasource import Datasource
+from great_expectations.datasource.fluent.config_str import ConfigStr
+from tests.test_utils import load_data_into_test_database
 
 yaml = YAMLHandler()
 
+"""
+# <snippet name="tests/integration/docusaurus/setup/configuring_data_contexts/how_to_configure_credentials.py export_env_vars">
+export MY_DB_PW=password
+export POSTGRES_CONNECTION_STRING=postgresql://postgres:${MY_DB_PW}@localhost:5432/postgres
+# </snippet>
+"""
+
 """
 # <snippet name="tests/integration/docusaurus/setup/configuring_data_contexts/how_to_configure_credentials.py config_variables_yaml">
-my_postgres_db_yaml_creds:
-  drivername: postgresql
-  host: localhost
-  port: 5432
-  username: postgres
-  password: ${MY_DB_PW}
-  database: postgres
+my_postgres_db_yaml_creds: postgresql://localhost:${MY_DB_PW}@$localhost:5432/postgres
 # </snippet>
 """
 
+# NOTE: The following code is only for testing and can be ignored by users.
```

Comment:
Do we need this comment now under FDS?

Reply:
good call, I dont think we need it in the Snippet world.